src/main/java/io/pravega/connectors/flink/AbstractStreamingReaderBuilder.java

```diff
@@ -109,6 +112,16 @@ public B withEventReadTimeout(Time eventReadTimeout) {
         return builder();
     }
 
+    /**
+     * Configures the maximum outstanding checkpoint request to Pravega.
```

Comment:
I think it is worth elaborating on this parameter and how it affects the behavior of the connector, specifically, we should say what happens when we try to request more than the specified maximum.

Reply:
fixed